src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,114 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.UserInvalidException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final long ID = 1;
+    private static final String LOGIN = "someLogin";
+    private static final String PASSWORD = "password";
+    private static final String INVALID_PASSWORD = "pas";
+    private static final int AGE = 20;
+    private static final int INVALID_AGE = 12;
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setId(ID);
+        user.setAge(AGE);
+        user.setLogin(LOGIN);
+        user.setPassword(PASSWORD);
+    }
+
+    @AfterEach
+    void clearStorage() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_user_isOk() {
+        assertEquals(user, registrationService.register(user));
+    }
+
+    @Test
+    void register_userNull_isNotOk() {
+        User userNull = null;
+        Assertions.assertThrows(UserInvalidException.class, () -> {
```

Comment:
```suggestion
        assertThrows(UserInvalidException.class, () -> {
```
likewise in the following cases